openssl/src/ssl/mod.rs

```diff
@@ -1434,6 +1438,56 @@ impl<S> SslStream<S> {
         }
     }
 
+    /// Gets the next DTLS handshake timeout.
+    ///
+    /// This is intended to be used with a nonblocking socket, where the value
+    /// can be used to set timeouts on reading and performing handshakes.
+    ///
+    /// `handle_dtlsv1_timeout` should be called when the timeout expires to
+    /// ask OpenSSL to retransmit.
```

Comment:
Can you note in the documentation that this method and the one below requires the dtls features?